podcasts/AppDelegate.swift

```diff
@@ -296,7 +296,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     private func updateRemoteFeatureFlags() {
-        #if !DEBUG
+        guard !BuildEnvironment.current.hasDebugFlag else { return }
```

Comment:
I believe we can do:

```suggestion
        guard BuildEnvironment.current != .debug else { return }
```

Then that's all we need (and we can delete the `hasDebugFlag`! Wdyt?

Reply:
Sounds good to me! As long as the `STAGING` flag isn't really used for anything special.

Reply:
Nope, it's not! We just use it when testing locally in `Staging` (and we have plans on changing that to be a toggle in-app because it makes Xcode to re-build the whole app)